st2client/st2client/models/core.py

```diff
@@ -507,12 +507,11 @@ class AsyncRequest(Resource):
 
 class PackResourceManager(ResourceManager):
     @add_auth_token_to_kwargs_from_env
-    def install(self, packs, force=False, python3=False, skip_dependencies=False, **kwargs):
+    def install(self, packs, force=False, skip_dependencies=False, **kwargs):
```

Comment:
Keep the `python3` argument here, but emit a warning that it's now ignored, and then don't pass it in the `payload` (like you already do). This will keep backwards compatibility for the CLI.